gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/membership/query_service/MembershipQueryService.java

```diff
@@ -22,4 +22,6 @@
 public interface MembershipQueryService {
     Collection<Membership> findByReferenceAndRoleId(Membership.ReferenceType referenceType, String referenceId, String roleId);
     Collection<Membership> findByReferencesAndRoleId(Membership.ReferenceType referenceType, List<String> referenceIds, String roleId);
+
+    Collection<Membership> findGroupsUserBelong(String userId);
```

Comment:
```findGroupsUserBelongsTo```?

Reply:
--> `findGroupsByUserId` ?

Reply:
@jourdiw In membership, there is not "userId" it is `memberId` and it matches to a "userId" only when the `memberType` is `USER`.

This method intends to find all the "groups" to which a user belongs: it is returning all membership having a referenceType `GROUP` where the `memberId` is the user. 

I prefer `findGroupsUserBelongsTo` because it well summarizes the behavior even if we don't return actual groups 🙈 However, if it is not english enough, I'm opened to suggestion 😅

Reply:
`findGroupsThatUserBelongsTo` 👍 🇬🇧 🇺🇸